docker-compose-windows-internal.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 The Cobalt Authors. All Rights Reserved.
```

Comment:
The copyright year represents when the file was created, we don't want to change it